### PR TITLE
[4.0] Media Action plugins

### PR DESCRIPTION
**Pull request by brianteeman**
This PR does the following

1. removes unused strings
2. changes labelclass="d-none" to hiddenLabel in the xml
3. adds missing labels (even if they are not visible)

ToDo in a later PR change the radio buttons in rotate to real buttons in a button group. The current code breaks accessibility but thats for another PR

### CODE REVIEW

File: administrator/language/en-GB/plg_media-action_resize.ini [hunk 1]
```diff
@@ -6,12 +6,12 @@
 PLG_MEDIA-ACTION_RESIZE="Media Action - Resize"
 PLG_MEDIA-ACTION_RESIZE_BATCH_DESC="Settings for server side actions when images are created."
 PLG_MEDIA-ACTION_RESIZE_BATCH_LABEL="Batch Settings"
-PLG_MEDIA-ACTION_RESIZE_BATCH_MAX_HEIGHT_DESC="The maximum height of an image to resize. When empty, no resizing is performed."
```
Comment (Quy) [inline administrator/language/en-GB/plg_media-action_resize.ini:9]: Please delete in `\plugins\media-action\resize\resize.xml`.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 0a7ea69c62f5270fbe08003f3d2979befd4a77b0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29970">issues.joomla.org/tracker/joomla-cms/29970</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0a7ea69c62f5270fbe08003f3d2979befd4a77b0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29970">issues.joomla.org/tracker/joomla-cms/29970</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29970">issues.joomla.org/tracker/joomla-cms/29970</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**